crt_portal/static/sass/_uswds-theme-typography.scss

```diff
@@ -420,3 +424,69 @@ $theme-lead-font-family:           'heading';
 $theme-lead-font-size:             'lg';
 $theme-lead-line-height:           6;
 $theme-lead-measure:               6;
+
+
+// Custom type variables -- note that these are not USWDS-native variables
+$theme-heading-font-size-xl:       2.8125rem; // 45px - h1 display
+$theme-heading-font-size-lg:       2rem; // 32px - h2 display
+$theme-heading-font-size-md:       1.375rem; // 22px - h3 display
+$theme-heading-font-size-sm:       1rem; // 16px - h4 display
+$theme-heading-font-size-lg-mobile:1.5rem; // 24px - mobile headers
+$theme-heading-font-size-md-mobile:1.25rem; // 20px - mobile headers
+$theme-text-font-size-lg:          1.125rem; // 18px - lead body copy
+$theme-text-font-size-md:          1rem; //16px - regular body copy
+$theme-text-font-size-sm:          0.875rem; // 14px - small body copy
+$theme-text-font-size-xs:          0.75rem; // 12px - accent text
+
+$theme-body-copy-line-height:      1.625;
+
+// Custom type mixins
+@mixin text--body-copy__xsmall {
+  font-size: $theme-text-font-size-xs;
+  line-height: 1.3;
+}
+
+@mixin text--body-copy__small {
+  font-size: $theme-text-font-size-sm;
+  line-height: 1.5;
+}
+
+@mixin text--body-copy__medium {
+  font-size: $theme-text-font-size-md;
+  line-height: 1.625;
+}
+
+@mixin text--body-copy__large {
+  font-size: $theme-text-font-size-lg;
+  line-height: 1.5;
+}
+
+@mixin text--heading__small {
+  font-size: $theme-heading-font-size-sm;
+  line-height: 1.5;
+}
+
+@mixin text--heading__medium {
+  font-size: $theme-heading-font-size-md;
+  line-height: 1.5;
+  @include at-media-max(mobile-lg) {
+    font-size: $theme-heading-font-size-md-mobile;
+    line-height: 1.3;
+  }
+}
+
+@mixin text--heading__large {
+  font-size: $theme-heading-font-size-lg;
+  line-height: 1.3;
+  @include at-media-max(mobile-lg) {
+    font-size: $theme-heading-font-size-lg-mobile;
+  }
+}
+
+@mixin text--heading__xlarge {
+  font-size: $theme-heading-font-size-xl;
+  line-height: 1.3;
+  @include at-media-max(mobile-lg) {
+    font-size: $theme-heading-font-size-lg;
+  }
+}
```

Comment:
I don't entirely love putting custom code in the `_uswds-theme` files, but those are also the first files I'd look at if I were trying to figure out what was going on. I'm open to putting these somewhere else if that makes sense to other folks

Reply:
I think this is a good place :+1: